actions-support/simulate_ci_image_build.sh

```diff
@@ -8,14 +8,15 @@ cd $(dirname $0)
 # Assume that the dist step has already been run, and copy those file here so
 # they're available for the buildx build context.
 mkdir -p ./ci-image/dist
-cp ../dist/dbt-toolchain-*-linux*.tar.gz ./ci-image/dist/
+rsync -cv ../dist/dbt-toolchain-*-linux*.tar.gz ./ci-image/dist/
 
 # Run the build
 docker buildx build \
   --platform linux/amd64 \
   --load \
-  -t deluge-ci-image:latest \
+  -t deluge-ci-image:v$(cat ../VERSION) \
```

Comment:
I've removed the VERSION file from the repo, it's now a part of the config, so I'm unsure what to do about resolving the version number

Reply:
I'll just revert this part of the change, it only matters for using this script locally.

Reply:
I've added a version.py python file to extract the version from the config, so I'll just merge this and fix as necessary.